src/services/codefixes/helpers.ts

```diff
@@ -304,6 +306,31 @@ export function addNewNodeForMemberSymbol(
         if (method) addClassElement(method);
     }
 
+
+    function createModifiers(): NodeArray<Modifier> | undefined {
+        let modifiers: Modifier[] | undefined;
+
+        if (modifierFlags) {
+            modifiers = combine(modifiers, factory.createModifiersFromModifierFlags(modifierFlags));
+        }
+
+        if (shouldAddOverrideKeyword()) {
+            modifiers = append(modifiers, factory.createToken(SyntaxKind.OverrideKeyword));
+        }
+
+        return modifiers && factory.createNodeArray(modifiers);
+    }
+
+    function shouldAddOverrideKeyword(): boolean {
+        if (!context.program.getCompilerOptions().noImplicitOverride || !enclosingDeclaration.heritageClauses) {
+            return false;
+        }
+
+        const valueDeclaration = checker.getSymbolAtLocation(enclosingDeclaration.heritageClauses[0].types[0].expression)?.valueDeclaration;

```

Comment:
This isn't yet correct per https://github.com/microsoft/TypeScript/pull/51033#discussion_r1045290270 - we should also check whether the specific method being implemented is `abstract`. Will get to this ... soon!

Reply:
This was actually simpler to implement 😄